__tests__/app.test.js

```diff
@@ -339,6 +339,37 @@ describe("/api/articles/:article_id/comments", () => {
     });
 });
 
+describe("/api/comments/:comment_id", () => {
+    test("DELETE204: successfully deletes comment by id, returning no content", () => {
+        return request(app)
+            .delete("/api/comments/5")
+            .expect(204)
+            .then(({ body }) => {
+                expect(body).toEqual({});
```

Comment:
This isn't needed. You can just expect a 204 from this and finish the test there.